### PR TITLE
playwright form 2[textarea not working]

### PR DESCRIPTION
**Pull request by tarunmun12**
Added page locators for _selectRice, _selectQA and _textarea.
textarea not working (for the moment)

### CODE REVIEW

File: playwrightForm2/PlaywrightTests/UnitTest1.cs [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+            await Page.Locator(_selectQA).SelectOptionAsync("Front"); //added
+
+            // type in textarea
+            await Page.Locator(_textarea).FillAsync("dummy text"); //added 
+
+            // click on checkbox
+            await Page.Locator(_checkBox).ClickAsync();
+
+            //  let's check if all good
+            await Page.PauseAsync();
```
Comment (b10nde1) [inline playwrightForm2/PlaywrightTests/UnitTest1.cs]: can remove the pause... this has been used to check on local only and no longer needed I think

File: playwrightForm2/PlaywrightTests/UnitTest1.cs [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+            // check text title and description
+            await Expect(Page.Locator(_textTitle)).ToHaveTextAsync(textTitle);
+            await Expect(Page.Locator(_textDescription)).ToHaveTextAsync(textDescription);
+
+            // type email and password
+            await Page.Locator(_email).FillAsync(GetGenericData(emailAddress));
+            await Page.Locator(_password).FillAsync(GetGenericData(password));
+
+            // select rice
+            await Page.Locator(_selectRice).SelectOptionAsync("Brown rice"); //added
```
Comment (b10nde1) [inline playwrightForm2/PlaywrightTests/UnitTest1.cs]: do not hardcode testdata

File: playwrightForm2/PlaywrightTests/UnitTest1.cs [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+            // select QA
+            await Page.Locator(_selectQA).SelectOptionAsync("Front"); //added
```
Comment (b10nde1) [inline playwrightForm2/PlaywrightTests/UnitTest1.cs]: do not hardcode testdata

File: playwrightForm2/PlaywrightTests/UnitTest1.cs [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+            // type in textarea
+            await Page.Locator(_textarea).FillAsync("dummy text"); //added
```
Comment (b10nde1) [inline playwrightForm2/PlaywrightTests/UnitTest1.cs]: do not hardcode testdata

File: playwrightForm2/PlaywrightTests/UnitTest1.cs (hunk 1 above)
Comment (tarunmun12) [reply playwrightForm2/PlaywrightTests/UnitTest1.cs]: Changes done

File: playwrightForm2/PlaywrightTests/UnitTest1.cs (hunk 2 above)
Comment (tarunmun12) [reply playwrightForm2/PlaywrightTests/UnitTest1.cs]: Changes done

File: playwrightForm2/PlaywrightTests/UnitTest1.cs (hunk 3 above)
Comment (tarunmun12) [reply playwrightForm2/PlaywrightTests/UnitTest1.cs]: Changes done

File: playwrightForm2/PlaywrightTests/UnitTest1.cs (hunk 4 above)
Comment (tarunmun12) [reply playwrightForm2/PlaywrightTests/UnitTest1.cs]: Changes done

**Pull request closed without merging**